src/components/header/Header.tsx

```diff
@@ -1,4 +1,91 @@
+              fill="#1D2026"
+              fill-rule="nonzero"
+            />
+          </svg>
+          <div className={style['menu-items']}>
+            {' '}
+            {menuItems.map((item) => (
+              <p className={style['items']}>{item}</p>
+            ))}
+          </div>
+
+          <ModalWindow
+            isOpen={isModalOpen}
+            onClose={() => setIsModalOpen(false)}
+          >
+            <ul>
+              {menuItems.map((item, index) => (
+                <li key={index}>{item}</li>
+              ))}
+            </ul>
+          </ModalWindow>
+        </div>
+        <div className={style['header-profile']}>
+          <svg
+            onClick={() => handleCartClick()}
```

Comment:
You don't need to create an anonymous function here. Just use the link to the function by its name.